### ISSUE TITLE
Not able to Communicate with the internal Kubernetes SVC from Openshift newly added Node

### ISSUE THREAD

**Issue by naidu571** [openshift/origin#21651]
[provide a description of the issue]
The newly scaleup node to the existing Openshify Cluster not able to communicate with the internal cluster services and gets the below output says that No Route to host.

[root@i ~]# curl https://172.30.0.1:443/api/v1
curl: (7) Failed connect to 172.30.0.1:443; No route to host

##### Version
[provide output of the `openshift version` or `oc version` command]

oc v1.5.1+7b451fc
kubernetes v1.5.2+43a9be4
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server : https://******:8443
openshift v3.7.2+cd74924-1
kubernetes v1.7.6+a08f5eeb62

##### Steps To Reproduce
1. Run the playbook to scaleup and new node to the cluster.
2.Run the curl command to get internal Kubernetes API

##### Additional Information
[try to run `$ oc adm diagnostics` (or `oadm diagnostics`) command if possible]
[if you are reporting issue related to builds, provide build logs with `BUILD_LOGLEVEL=5`]
[consider attaching output of the `$ oc get all -o json -n <namespace>` command to the issue]
[visit https://docs.openshift.org/latest/welcome/index.html]

[Note] Determining if client configuration exists for client/cluster diagnostics
[Note] No client configuration specified; skipping client and cluster diagnostics.
[Note] Performing systemd discovery

[Note] Running diagnostic: AnalyzeLogs
       Description: Check for recent problems in systemd service logs
       
Info:  Checking journalctl logs for 'origin-node' service
Info:  Checking journalctl logs for 'docker' service

[Note] Skipping diagnostic: MasterConfigCheck
       Description: Check the master config file
       Because: No master config file was detected
       
[Note] Running diagnostic: NodeConfigCheck
       Description: Check the node config file
       
Info:  Found a node config file: /etc/origin/node/node-config.yaml

[Note] Running diagnostic: UnitStatus
       Description: Check status for related systemd units
       
[Note] Summary of diagnostics execution (version v3.7.2+cd74924-1):
[Note] Completed with no errors or warnings seen.

**Comment by naidu571** [comment]
Here is the more information on the issue. 
Observed the mtu value to 1500 for tun0 interface & Failed to execute iptables-restore: exit status 1 (iptables-restore: invalid option -- '5 on the journald logs.

**Comment by liuyue2656** [comment]
I have the same issue.

I install OpenShift with [openshift-ansible](https://github.com/openshift/openshift-ansible), branch: openshift-ansible-3.7-fix : https://github.com/openshift/openshift-ansible/tree/openshift-ansible-3.7-fix

Install program is success, but my origin-nodes is cannot Ready with an Error Message:
```
origin-node: E0115 16:38:08.910714   31880 proxier.go:1601] Failed to execute iptables-restore: exit status 1 (iptables-restore: invalid option -- '5'
origin-node: Try `iptables-restore -h' for more information.
origin-node: )
```

Then I run `oc get pods` get the pods status is ERROR:
```
$ oc get pod
NAME                        READY     STATUS    RESTARTS   AGE
docker-registry-1-deploy    0/1       Pending   0          21h
registry-console-1-deploy   0/1       Error     0          21h
router-2-deploy             0/1       Pending   0          20h
```

Are you fixed it?

**Comment by imcsk8** [comment]
This issue has been fixed in OKD 3.7.72: https://github.com/kubernetes/kubernetes/issues/58956#issuecomment-444511729

Feel free to reopen this bug if the problem persists.
